src/treasuryDAO/types.mligo

```diff
@@ -29,3 +29,8 @@ type treasury_contract_extra =
   * tez // min_xtz_amount
   * tez // max_xtz_amount
   )
+
+type initial_treasuryDAO_storage =
+  { trivial_data : initial_data
+  ; contract_extra : treasury_contract_extra
+  }
```

Comment:
I wonder why you went with a different approach from `registryDAO` here, the `treasury_contract_extra` only use was for configuration, I think it should now be removed in favor of a single:
```suggestion
type initial_treasuryDAO_storage =
  { base_data : initial_data
  ; frozen_scale_value : nat
  ; frozen_extra_value : nat
  ; max_proposal_size : nat
  ; slash_scale_value : nat
  ; slash_division_value : nat
  ; min_xtz_amount : nat
  ; max_xtz_amount : nat
  }
```

Reply:
I looked at the function definition an tried to use the same value as in definition. There we have all this data passed as one big tuple

Reply:
Oh yeah, my suggestion is to update the definition as well in a way that's similar to registryDAO